content/applications/general/users/access_rights.rst

```diff
@@ -2,66 +2,184 @@
 
-.. image:: access_rights/groups-record-rules.png
+To modify existing groups, click on an existing group from the list displayed on the
+:guilabel:`Groups` page, and edit the contents of the form.
+
+Enter a :guilabel:`Name` for the group and tick the checkbox next to :guilabel:`Share Group`, if
+this group was created to set access rights for sharing data with some users.
+
+.. important::
+   Always test the settings being changed to ensure they are being applied to the correct users.
+
+The group form contains multiple tabs for managing all elements of the group. In each tab, click
+:guilabel:`Add a line` to add a new row for users or rules, and click the :guilabel:`❌` icon to
+remove a row.
+
+.. image:: access_rights/groups-form.png
    :align: center
-   :alt: View of a group’s form emphasizing the tab record rules in Odoo
+   :alt: Click each tab in the groups form to modify the settings of the group.
+
+- :guilabel:`Users` tab: lists the current users in the group. Users listed in black have
+  administrative rights. Users without administrative access appear in blue. Click :guilabel:`Add a
+  line` to add *users* to this group.
+- :guilabel:`Inherited` tab: inherited means that users added to this group are automatically added
+  to the groups listed on this tab. Click :guilabel:`Add a line` to add *inherited groups*.
+
+  .. example::
+     For example, if the group *Sales/Administrator* lists the group *Website/Restricted Editor* in
+     its :guilabel:`Inherited` tab, then any users added to the *Sales/Administrator* group
+     automatically receive access to the *Website/Restricted Editor* group, as well.
+
+- :guilabel:`Menus` tab: defines which menus/models the group can have access to. Click
+  :guilabel:`Add a line` to add a specific *menu*.
+- :guilabel:`Views` tab: lists which views in Odoo the group has access to. Click :guilabel:`Add a
+  line` to add a *view* to the group.
+- :guilabel:`Access Rights` tab: lists the first level of rights (models) that this group has access
+  rights to. Click :guilabel:`Add a line` to link *access rights* to this group. In this tab, the
+  :guilabel:`Model` column represents the common name of the menu/model, and the :guilabel:`Name`
+  column represents the technical name given to the model. For each model, enable the following
+  options as appropriate:
+
+  - :guilabel:`Read`: users can see the object's existing values.
+  - :guilabel:`Write`: users can edit the object's existing values.
+  - :guilabel:`Create`: users can create new values for the object.
+  - :guilabel:`Delete`: users can delete values for the object.
+
+  .. tip::
+     First try searching for the common name of the model in the drop-down menu of the
+     :guilabel:`Model` column. The :guilabel:`Model` technical name can be found by expanding the
+     model common name, which can be done by clicking the :guilabel:`expand` button, represented by
+     an :guilabel:`external link` icon.
```

Comment:
There is no "expand" text on the UI, so no :guilabel:, can italicize since you are introducing an alias name for the **External Link** button 
```suggestion
     model common name, which can be done by clicking the *expand* button, represented by an
     :guilabel:`(external link)` icon.
```
or preferably...
```suggestion
     model common name, which can be done by clicking the :guilabel:`(external link)` icon.
```